### PR TITLE
Make voluntary exits release-ready

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

This PR wraps up voluntary exits:
- Final confirmation step links the user to the appropriate page and anchor in the docs portal
- Final confirmation step requires the passphrase from the docs portal
- The `accounts-v2 exit` command triggers the real implementation

**Which issues(s) does this PR fix?**

Part of #6882

**Other notes for review**

The documentation PR has not been merged yet: https://github.com/prysmaticlabs/documentation/pull/227

### CODE REVIEW

File: shared/promptutil/validate.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ const (
 	minPasswordScore = 2
 )
 
+const incorrectPhrase = "input does not match wanted phrase"
```
Comment (terencechain) [inline shared/promptutil/validate.go:20]:
```suggestion
var incorrectPhrase = errors.New("input does not match wanted phrase")
```

File: shared/promptutil/validate.go [hunk 2]
```diff
@@ -111,3 +113,12 @@ func ValidatePasswordInput(input string) error {
 	}
 	return nil
 }
+
+// ValidatePhrase checks whether the user input is equal to the wanted phrase. The verification is case sensitive.
+func ValidatePhrase(input string, wantedPhrase string) error {
+	trimmedInput := strings.Join(strings.Fields(strings.TrimSpace(input)), " ")
+	if trimmedInput != wantedPhrase {
+		return errors.New(incorrectPhrase)
```
Comment (terencechain) [inline shared/promptutil/validate.go:121]:
```suggestion
		return incorrectPhrase
```

**Review by terencechain — COMMENTED** [review]
more idiomatic for error

File: shared/promptutil/validate.go [hunk 3]
```diff
@@ -111,3 +113,12 @@ func ValidatePasswordInput(input string) error {
 	}
 	return nil
 }
+
+// ValidatePhrase checks whether the user input is equal to the wanted phrase. The verification is case sensitive.
+func ValidatePhrase(input string, wantedPhrase string) error {
+	trimmedInput := strings.Join(strings.Fields(strings.TrimSpace(input)), " ")
```
Comment (prestonvanloon) [inline shared/promptutil/validate.go:119]: why fields + join? Is it invalid to have 2 spaces in the middle of a password?

File: validator/accounts/v2/cmd_accounts.go [hunk 4]
```diff
@@ -126,7 +126,7 @@ this command outputs a deposit data string which is required to become a validat
 			},
 		},
 		{
-			Name:        "voluntary-exit",
```
Comment (nisdas) [inline validator/accounts/v2/cmd_accounts.go:129]: previous name was better, its more in line with the spec

File: validator/accounts/v2/accounts_exit.go [hunk 5]
```diff
@@ -158,10 +151,12 @@ func interact(cliCtx *cli.Context, r io.Reader, validatingPublicKeys [][48]byte)
...
 		"Please navigate to the following website and make sure you understand the current implications " +
 		"of a voluntary exit before making the final decision:"
-	promptURL := au.Blue("https://docs.prylabs.network/docs/faq/#can-i-get-back-my-testnet-eth-how-can-i-withdraw-my-validator-gains")
-	promptQuestion := "Do you still want to continue with the voluntary exit? Y/N"
+	promptURL := au.Blue("https://docs.prylabs.network/docs/wallet/nondeterministic/exiting-a-validator/#withdrawal-delay-warning")
+	promptQuestion := "If you still want to continue with the voluntary exit, please input the passphrase from the above URL"
 	promptText := fmt.Sprintf("%s\n%s\n%s\n%s", promptHeader, promptDescription, promptURL, promptQuestion)
-	resp, err := promptutil.ValidatePrompt(r, promptText, promptutil.ValidateYesOrNo)
+	resp, err := promptutil.ValidatePrompt(r, promptText, func(input string) error {
+		return promptutil.ValidatePhrase(input, "Exit my validator")
```
Comment (nisdas) [inline validator/accounts/v2/accounts_exit.go:158]: can we make this phrase a declared constant and just use it in the accounts code and tests.

**Review by nisdas — APPROVED** [review]

**Pull request merged**